cdap-watchdog/src/test/java/co/cask/cdap/logging/write/LogCleanupTest.java

```diff
@@ -235,18 +235,18 @@ public void testDeleteEmptyDir2() throws Exception {
 
     LogCleanup logCleanup = new LogCleanup(null, baseDir, RETENTION_DURATION_MS);
 
-    logCleanup.deleteEmptyDir("ns/" + logBaseDir, baseDir);
+    logCleanup.deleteEmptyDir("namespaces" + "/ns/" + logBaseDir, baseDir);
     // Assert base dir exists
     Assert.assertTrue(baseDir.exists());
 
     baseDir.mkdirs();
     // Assert root exists
     Assert.assertTrue(baseDir.exists());
-    logCleanup.deleteEmptyDir("ns/" + logBaseDir, baseDir);
+    logCleanup.deleteEmptyDir("namespaces" + "/ns/" + logBaseDir, baseDir);
```

Comment:
don't hardcode this here. can you use Namespaces util method? at the very minimum, get it from con?